src/js/table-of-contents/table-of-contents.js

```diff
@@ -27,55 +33,55 @@ var tableOfContents = function (content, target, options) {
   //
   // Methods
   //
-	/**
-	 * Merge user options into defaults
-	 * @param  {Object} obj The user options
-	 */
```

Comment:
Apologies for lines like this. Despite having editorconfig installed and the presence of a valid editorconfig file, tabs and spaces were still being mixed. I normalized to the dominant indentation used in the file (two spaces).

Reply:
S'okay. I have a housekeeping PR I've been working on locally to handle some stuff like this, so don't stress too much about it.